### ISSUE TITLE
Question about real world images

### ISSUE THREAD

**Issue by FantasyJXF** [huochaitiantang/pytorch-deep-image-matting#19]
I trained the stage 0 and the result in test dataset was good.

But when I try to use my own image for inference, it was far from what I expected.

I think it's because of the ** TRIMAP ** .

The trimap Adobe provide was very precise, how can we get such nice trimaps of ourself images?

**Comment by myCigar** [comment]
Hello, I want to know how the learning rate setting when you retrain.
